frontend/common/src/components/SearchRequestFilters/SearchRequestFilters.tsx

```diff
@@ -193,7 +431,6 @@ const SearchRequestFilters: React.FunctionComponent<
     },
   );
   const typeOfOpportunity = ""; // TODO: Replace with data fetched from api
```

Comment:
'typeOfOpportunity' is assigned a value but never used.eslint@typescript-eslint/no-unused-vars

Reply:
Oh ya I believe this got deprecated. Thanks 👍